### DISCUSSION TITLE
Integrate SEB in my quiz app

### DISCUSSION THREAD

**Discussion by iamdimitarkolev7** [SafeExamBrowser/seb-win-refactoring#565] [Q&A]
Hello,

What is the correct way to achieve transfering to safe exam browser when clicking a button for starting a quiz?
I used the following link: seb://?url=./test.php?testName=${testName}&config=./SebClientSettings.seb` but somethings wrong with the configuration file i guess. Can anybody help me?

Thanks in advance!

**Comment by danschlet** [comment]
The seb:// or better use sebs:// (which uses https by default) has to directly or indirectly link to a SEB configuration file (when SEB loads that URL, it waits for the server to initiate the download of a .seb config file).

If that doesn't work in your case, you should look at what exactly happens (is there a SEB error message?) or look at SEB log files.

PS: In general you would use SEB configuration files saved with the option "for starting an exam" (your config file name looks like the file might have been saved for the purpose "configuring client", see [manual](https://safeexambrowser.org/windows/win_usermanual_en.html#ConfigFilePane)).
